src/utils/utils.ts

```diff
@@ -0,0 +1,9 @@
+export const sortByYear = (a: any, b: any) => {
```

Comment:
it's better to make an explicit (movieA, movieB) names for parameters and type it that each parameter have `year` property at least